docs/turbo/generators/utils.ts

```diff
@@ -0,0 +1,64 @@
+import fetch from "node-fetch";
+
+interface Answers extends Object {
+  turboStars: string;
+  turboDownloads: string;
+  turboYearsSaved: string;
+}
+
+const PUBLIC_TB_TOKEN =
+  "p.eyJ1IjogIjAzYzA0Y2MyLTM1YTAtNDhhNC05ZTZjLThhMWE0NGNhNjhkZiIsICJpZCI6ICJmOWIzMTU5Yi0wOTVjLTQyM2UtOWIwNS04ZDZlNzIyNjEwNzIifQ.A3TOPdm3Lhmn-1x5m6jNvulCQbbgUeQfAIO3IaaAt5k";
+
+export async function releasePostStats(answers: Answers): Promise<string> {
+  const [starsResponse, downloadsResponse, timeSavedResponse] =
+    await Promise.all([
+      fetch("https://api.github.com/repos/vercel/turbo"),
+      fetch("https://api.npmjs.org/versions/turbo/last-week"),
+      fetch(
+        `https://api.us-east.tinybird.co/v0/pipes/turborepo_time_saved_ticker.json?token=${PUBLIC_TB_TOKEN}`
+      ),
```

Comment:
Just for you @mehulkar 😉

Reply:
YESSSSSSSSS. Next up we drop `jest` for `node:test`